src/main/java/Event.java

```diff
@@ -0,0 +1,43 @@
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.sql.Timestamp;
+
+class Event extends Task {
```

Comment:
Similar to Deadline class, this class needs access modifier for the constructors and Javadoc comment.